i5k/settings.py

```diff
@@ -395,14 +395,14 @@
                 'blast/css/codemirror.css',
                 'blast/css/xq-light.css',
                 'blast/css/kendo.common-bootstrap.core.css',
-                'blast/css/kendo.bootstrap.min.css',
-                'blast/css/jquery-ui.min.css',
-                'blast/dataTables/css/jquery.dataTables.min.css',
-                'blast/dataTables/css/dataTables.scroller.min.css',
-                'blast/dataTables/css/dataTables.colReorder.min.css',
+                'blast/css/kendo.bootstrap.css',
```

Comment:
looks like this is also going to remove some of the JS that is getting loaded multiple times.

Reply:
Yes, but it only happens in f6a21d2ce6cf4d37c781bf3200b3cdee1912bd0e.

Reply:
Right, pipeline also had the same file into a couple of different sections which made me think that (for example) underscore.js was getting bundled into app.min.js and blast.min.js  

I like that the duplicate libraries are getting removed.  This should help slim down what gets downloaded on the client side.

Reply:
Yep. This would help us slim down the repo size (well ... reduce the increasing rate of size).
We also can update the documents to indicate user to use `git clone --depth=50` to download only a slice of the repo.

Reply:
I'm surprised to see dataTables is completely removed,  I thought that was driving a lot of the functionality in the bottom left quadrent of the blast-results page.

Reply:
I think there is a confusion of the path. The datatable is saved outside the **js/** or **css/** folder.

Reply:
FTW, currently, if I `git clone --depth 50 https://github.com/NAL-i5K/genomics-workspace`. The size of the directories will be **131 MB**.

Reply:
With complete history copy, `git clone https://github.com/NAL-i5K/genomics-workspace` will need **541MB**.